docs/advanced_topics/api/v2/extra_usage.rst

```diff
@@ -0,0 +1,86 @@
+=====================================
+Extra information on exposing the API
+=====================================
+
+Images
+------
+
+If you have in your model an image you can expose the image by using the ImageRenditionField.
+:class:`wagtail.images.api.fields.ImageRenditionField`. The ImageRenditionField will return url, width, height.
+
+..code-block:: python
+
+    class SomePage(Page):
+        ...
+        image = models.ForeignKey('wagtailimages.Image', null=True, blank=True, on_delete=models.SET_NULL, related_name='+')
+        ...
+
+        @property
+        def img(self):
+            return ImageRenditionField('width-700').to_representation(self.image)
+
+        @property
+        def alt(self):
+            return self.image.title
```

Comment:
Here and above, if the self.image is not present (eg. user has 'cleared' that selection), the API will not return a key for that item at all.

It might be worth mentioning, ideally by example, that this will happen and how to work around it, for example:

```
    @property
    def alt(self):
        # if no result exists, the API will not return a key
        # you may want to to ensure there is always a value returned
        if self.image:
            return self.image.title
        return ""
```